lib/ProteinStructureUtils/Utils/PDBUtils.py

```diff
@@ -185,8 +327,124 @@ def import_model_pdb_file(self, params):
+                 'name': mmcif_name,
+                 'data': data}]
+        })[0]
+        obj_ref = f"{info[6]}/{info[0]}/{info[4]}"
+
+        returnVal = {'structure_obj_ref': obj_ref}
+
+        if create_report:
+            report_output = self._generate_report('import_experiment_pdb_file', obj_ref,
+                                                  workspace_name, n_polypeptides,
+                                                  mmcif_name, file_path)
+            returnVal.update(report_output)
+
+        return returnVal
+
+    def batch_import_pdb_files(self, params):
+        """
+        batch_import_pdb_files: upload two sets of pdb files and convert into a
+                               KBaseStructure.ProteinStructures object
+        1. from the params['meta_file_path'] spreadsheet, sort out the model_pdb_paths,
+           exp_pdb_paths and the kbase_meta_data
+        2. call import_model_pdb_file on each entry in model_pdb_paths,
+           call import_experiment_pdb_file on each entry in exp_pdb_paths, and
+           call _parse_kbase_metadata to parse for KBase metadata
+        3. assemble the data for a ProteinStructures and save the data object
+        4. generate a report
+        """
+
+        (model_pdb_files, exp_pdb_files, workspace_name,
+         structures_name) = self._validate_batch_import_pdb_files_params(params)
+
+        if not isinstance(workspace_name, int):
+            workspace_id = self.dfu.ws_name_to_id(workspace_name)
+        else:
+            workspace_id = workspace_name
+
+        model_pdb_objects = list()
+        exp_pdb_objects = list()
+        successful_files = list()
+        failed_files = list()
+        protein_structures = dict()
+        total_structures = 0
+
+        # loop through the lists model_pdb_file_paths and exp_pdb_file_paths
+        if model_pdb_files:
+            for model_file in model_pdb_files():
+                model_pdb_ref = self.import_model_pdb_file(params, False)
+
+                if model_pdb_ref:
+                    model_pdb_objects.append(model_pdb_ref)
+                    successful_files.append(model_file)
+                    total_structures += 1
+                else:
+                    failed_files.append(model_file)
+
+        if exp_pdb_files:
+            for exp_file in exp_pdb_files():
+                exp_pdb_ref = self.import_experiment_pdb_file(params, False)
+
+                if exp_pdb_ref:
+                    exp_pdb_objects.append(exp_pdb_ref)
+                    successful_files.append(exp_file)
+                    total_structures += 1
+                else:
+                    failed_files.append(exp_file)
+
+        if model_pdb_objects:
+            protein_structures['model_structures'] = model_pdb_objects
+        if exp_pdb_objects:
+            protein_structures['experimental_structures'] = exp_pdb_objects
+        protein_structures['total_structures'] = total_structures
```

Comment:
you can just do `protein_structures['total_structures'] = len(successful_files)` here and completely remove `total_structures` var